human-tracking-and-pose-detection/main.py

```diff
@@ -0,0 +1,234 @@
+downloadModel=True
```

Comment:
If downloadModel is False, "tflite" models will fail to run for the first time. downloadModel shouldn't be a variable. The code should check if model requested by the user is already predownloaded, if not it should then download the model. 
Ideally, all models should be predownloaded before running the script at a user-defined path. The model path should be taken as command line argument. You can separate wget commands in a separate bash script.